PyDSS/defaults/simulation.toml

```diff
@@ -14,11 +14,8 @@
 # Active Scenario- [String] - Project scenario to use
 # DSS File- [String] - The main OpenDSS file
 [Project]
-"Start Year" = 2017
-"Start Day" = 1
-"Start Time (min)" = 0.0
-"End Day" = 1
-"End Time (min)" = 1439.0
+"Start time" = "1/1/2020 00:00:00"
+"Simulation duration (min)" = 1440.0
 "Date offset" = 0
```

Comment:
This line needs to be deleted. Causes a runtime error.

Reply:
Date offset replaced with '"Loadshape start time"' which should be a date time string that matched the default opendss timestamp format.